src/main/java/software/amazon/smithy/lsp/SmithyLanguageServer.java

```diff
@@ -125,8 +103,7 @@ public void exit() {
 
   @Override
   public WorkspaceService getWorkspaceService() {
-    // TODO Auto-generated method stub
-    return null;
+    return new SmithyWorkspaceService(this.tds);
```

Comment:
I was confused for a second why the notifications weren't being sent - this is why.